src/http/http_rpc_context.h

```diff
@@ -261,9 +263,19 @@ namespace http
       response_headers[std::string(name)] = value;
     }
 
-    virtual bool response_is_error() const override
+    virtual void set_apply_writes(bool apply) override
+    {
+      explicit_apply_writes = apply;
+    }
+
+    virtual bool get_apply_writes() const override
     {
-      return response_status != HTTP_STATUS_OK;
+      if (explicit_apply_writes.has_value())
+      {
+        return explicit_apply_writes.value();
+      }
+
+      return (response_status / 100) == 2;
```

Comment:
what does `response_status / 100 == 2` do? maybe add a comment?

Reply:
This checks whether the response_status is in the 200 range - have rewritten to be clearer and added a comment.

Reply:
HTTP error codes in [200, 300) indicate success, this checks for that. I agree a comment is warranted :)